roxie/ccd/ccdquery.cpp

```diff
@@ -88,7 +93,53 @@ class CQueryDll : implements IQueryDll, public CInterface
     CQueryDll(const char *_dllName, ILoadedDllEntry *_dll) : dllName(_dllName), dll(_dll)
     {
         StringBuffer wuXML;
-        if (!selfTestMode && getEmbeddedWorkUnitXML(dll, wuXML))
+        if (getEmbeddedGraphXML(dll, wuXML))
```

Comment:
The lost test of selTestMode means this will crash

Reply:
Fixed